notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1693 @@
+{
```

Comment:
<p>I would somehow try to summarize here also the cases when to using plain [] (which is: select one or more columns, filtering of the rows)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>